tests/schedule_test.py

```diff
@@ -3,15 +3,36 @@
 import httpretty
 import pygerduty.v2
 import textwrap
+import requests
+
+###################
+# Version 2 Tests #
+###################
 
 @httpretty.activate
-def test_get_schedule_v1():
+def test_get_schedule_v2():
     body = open('tests/fixtures/schedule_v2.json').read()
     httpretty.register_uri(
         httpretty.GET, "https://api.pagerduty.com/schedules/PI7DH85",
         body=body, status=200)
     p = pygerduty.v2.PagerDuty("contosso", "password")
     schedule = p.schedules.show("PI7DH85")
-    print schedule
+
     assert schedule.self_ == "https://api.pagerduty.com/schedules/PI7DH85"
     assert len(schedule.schedule_users) == 1
+    assert len(schedule.schedule_layers) == 1
+    assert schedule.schedule_layers[0].start == "2015-11-06T21:00:00-05:00"
+
+@httpretty.activate
+def test_list_schedules_v2():
+    body = open('tests/fixtures/schedule_list_v2.json').read()
+    httpretty.register_uri(
+	    httpretty.GET, "https://api.pagerduty.com/schedules",
+	    body=body, status=200)
+    p = pygerduty.v2.PagerDuty("contosso", "password")
+    schedules = []
+    for s in p.schedules.list():
```

Comment:
@gmjosack when I do this, the test goes into an infinite while loop in the list() method. Here is sample output when I put print statements in:
```
1
limit is: 25
offset is: 2486
[<Schedule: schedule_users=[<ScheduleUser: self_=u'https://api.pagerduty.com/users/PEYSGVF', id=u'PEYSGVF', type=u'user', html_url=u'https://subdomain.pagerduty.com/users/PEYSGVF', summary=u'PagerDuty Admin'>], name=u'Daily Engineering Rotation', html_url=u'https://subdomain.pagerduty.com/schedules/PI7DH85', summary=u'Daily Engineering Rotation', self_=u'https://api.pagerduty.com/schedules/PI7DH85', time_zone=u'America/New_York', escalation_policies=[<EscalationPolicy: self_=u'https://api.pagerduty.com/escalation_policies/PT20YPA', id=u'PT20YPA', type=u'escalation_policy_reference', html_url=u'https://subdomain.pagerduty.com/escalation_policies/PT20YPA', summary=u'Another Escalation Policy'>], type=u'schedule', id=u'PI7DH85', description=u'Rotation schedule for engineering'>]
1
limit is: 25
offset is: 2487
[<Schedule: schedule_users=[<ScheduleUser: self_=u'https://api.pagerduty.com/users/PEYSGVF', id=u'PEYSGVF', type=u'user', html_url=u'https://subdomain.pagerduty.com/users/PEYSGVF', summary=u'PagerDuty Admin'>], name=u'Daily Engineering Rotation', html_url=u'https://subdomain.pagerduty.com/schedules/PI7DH85', summary=u'Daily Engineering Rotation', self_=u'https://api.pagerduty.com/schedules/PI7DH85', time_zone=u'America/New_York', escalation_policies=[<EscalationPolicy: self_=u'https://api.pagerduty.com/escalation_policies/PT20YPA', id=u'PT20YPA', type=u'escalation_policy_reference', html_url=u'https://subdomain.pagerduty.com/escalation_policies/PT20YPA', summary=u'Another Escalation Policy'>], type=u'schedule', id=u'PI7DH85', description=u'Rotation schedule for engineering'>]
```

where 1 represents the length of the page, the limit is set to the default and the offset keeps getting incremented infinitely. I did notice that the response output changed for this between v1 and v2 but can't seem to figure out why its getting stuck in the loop and never did that before.